components/camel-mongodb/src/main/java/org/apache/camel/component/mongodb/converters/MongoDbBasicConverters.java

```diff
@@ -78,6 +81,21 @@ public static Document fromFileToDocument(File f, Exchange exchange) throws Exce
         return fromInputStreamToDocument(new FileInputStream(f), exchange);
     }
 
+    @Converter
+    public static Document fromByteArrayToDocument(byte[] data) {
```

Comment:
In some of the other converters we take in exchange as parameter, and use

ExchangeHelper.getCharsetName(exchange)

that defaults to utf-8

See for example IOConverter

Reply:
Will fix it

Reply:
done